circle.yml

```diff
@@ -31,7 +31,7 @@ deployment:
       - docker push soutech/champaign_web
       - ./deploy.sh $CIRCLE_SHA1 'champaign' 'champaign-production' 'champaign-assets-production' 'logs3.papertrailapp.com:44107'
   staging:
-    branch: development
+    branch: partial_cache
```

Comment:
this doesn't get merged. are you click-through testing on staging? is it working?